test/evp_test.c

```diff
@@ -111,6 +112,21 @@ static int memory_err_compare(EVP_TEST *t, const char *err,
     return r;
 }
 
+/* Options specific for evp test */
+typedef struct evp_test_settings_st {
+    unsigned char in_place;
+} EVP_TEST_SETTINGS;
```

Comment:
Why do we need a structure instead of just a value with either bit mask (in-place and not in-place)?

Reply:
I think the structure would be more readable if more options were possibly added in the future, rather than having multiple individual values, but only in this case that's not practically necessary.

Reply:
I don't think so. We have several modes and several options and the struct will only create a burden to maintain it. Several integer variables are better in this case

Reply:
Ok, thanks, fixed.